### DISCUSSION TITLE
How can I make the leading zeros of the ICD-9 code in the D_ICD_DIAGNOSES.csv file visible?

### DISCUSSION THREAD

**Discussion by DongHyukHwang** [MIT-LCP/mimic-code#1264] [MIMIC-III]
I downloaded the csv data from https://physionet.org/content/mimiciii-demo/1.4/. Both '01100' and '1100' are recorded in the ICD-9 code column as '1100'. I am wondering how can I make the leading '0' of '01100' visible.

**Comment by tompollard** [comment]
@DongHyukHwang The issue you are having is that your viewer (Excel?) is treating the field as a number, rather than a string. If you open the data in a text editor (e.g. Notepad++, Sublime, VS Code, Vim, etc) you should see the leading "0"s. The fix is to specify the correct data type (char/string) in your editor or code.
